### DISCUSSION TITLE
Images on Windows

### DISCUSSION THREAD

**Discussion by rodrigombsoares** [rdbende/Azure-ttk-theme#23] [Question] [answered]
Any idea on why the theme images doesn't show on Windows?
![check_tk](https://user-images.githubusercontent.com/38868277/152654994-9af6bc5e-977c-4821-9778-158a6983e010.png)
As example there is the checkbox that doesn't show as an image.

**Comment by rdbende** [comment, marked as answer]
Seems like you're using a `tk.Checkbutton`. Please change it to `ttk.Checkbutton`, since only ttk (themed tk) widgets can be themed.
